app.rb

```diff
@@ -12,9 +12,9 @@
 require 'net/http'
 
 # Load Environment Variables
-Prius.load(:gocardless_token)
-Prius.load(:gc_api_key_secret)
+Prius.load(:gc_access_token)
```

Comment:
I've renamed this to keep the `GC_` convention for environment variables related to GoCardless. You'll need to update the name of the variable on heroku, too.